[Bugfix:InstructorUI] Fix Notebook scrolling #12289

Aaditya-2407 opened the pull request:
![Uploading image (2).png…]()
### Why is this Change Important & Necessary?
Fixes #11096. Graders were unable to view the full student response in notebook codeboxes if the content overflowed vertically, making grading difficult.

### What is the New Behavior?
- Increased short answer codebox height to 120px.
- Enabled line wrapping to ensure all text remains visible.
- Forced a visible vertical scrollbar that bypasses layout clipping from parent containers.
- Added right padding to clear space for the scrollbar and prevent it from cutting off text.

### What steps should a reviewer take to reproduce or test the bug or new feature?
1. Open a gradeable using the Notebook view with a short answer component.
2. Input or view a response that exceeds 120px in height.
3. Verify the vertical scrollbar is visible and interactive.
4. Verify that text wraps correctly and is not obscured by the scrollbar.

### Automated Testing & Documentation
This is a UI/UX layout fix and was manually verified in the local Vagrant environment. No new automated tests were added.

### Other information
This is not a breaking change and does not include database migrations.

Aaditya-2407 committed:
Fix notebook scrolling: Enforce height, enable wrapping, and bypass layout clipping

JManion32 reviewed (changes requested):
Hello @Aaditya-2407 , thank you for your contribution! Please update this pull request to be more descriptive and follow Submitty conventions. Titles are limited to 40 characters, and start with a general descriptor which, in your case, would be [Bugfix:InstructorUI] (see the [Documentation / Title Check (pull_request)] test in GitHub actions).

In addition, please update your description using our template as a guide:

```
### Why is this Change Important & Necessary?
<!-- Include any GitHub issue that is fixed/closed using "Fixes #<number>" or "Closes #<number>" syntax.  
Alternately write "Partially addresses #<number>" or "Related to #<number>" as appropriate. -->

### What is the New Behavior?
<!-- Include before & after screenshots/videos if the user interface has changed. -->

### What steps should a reviewer take to reproduce or test the bug or new feature?

### Automated Testing & Documentation
<!-- Is this feature sufficiently tested by unit tests and end-to-end tests?  
If this PR does not add/update the necessary automated tests, write a new GitHub issue and link it below.  
Is this feature sufficiently documented on submitty.org?
Link related PRs or new GitHub issue to update documentation. -->

### Other information
<!-- Is this a breaking change?  
Does this PR include migrations to update existing installations?  
Are there security concerns with this PR? -->
```

**_These conventions are not optional._** They are important for keeping out pull requests consistent, descriptive, and easy to review. Thank you!

Aaditya-2407 commented:
Hi @JManion32 , thank you for the feedback! I have updated the PR title to follow the character limit and categorization format. I also updated the description using the Submitty template. Ready for further review!

JManion32 committed:
Merge branch 'main' into main

aconfo reviewed (changes requested):
<img width="907" height="280" alt="Image" src="https://github.com/user-attachments/assets/fdb0c92b-1860-4e7e-bfbf-ab44c21b9f36" /> I tested the change using notebook basic, and under the notebook tab, there is now 2 vertical bars, where the outer one has does not scroll, but the inner one does. Not only that, but the lower horizontal bar does scrolls when swiping to the side, but you cannot directly interact with it by clicking on it. I also noticed the frequent use of !important in the code, such as around line 155, which seems excessive.

Aaditya-2407 committed:
[Bugfix:InstructorUI] Refactor notebook scrolling to remove !important

Aaditya-2407 committed:
[Bugfix:InstructorUI] Clean refactor of notebook scrolling

aconfo reviewed (changes requested):
I only performed a functionality test. While testing, I noticed that when trying to submit an "example" to test, the textboxes were way bigger than necessary, as seen below. 

<img width="972" height="267" alt="Image" src="https://github.com/user-attachments/assets/ef4af202-a176-47af-bf1b-bfc4dc37d090" />

Furthermore, when reviewing the problem in Notebook, the boxes were all bigger, but I was not able to scroll, as seen here: 

<img width="906" height="242" alt="Image" src="https://github.com/user-attachments/assets/d1324a6a-1091-4367-bf83-ffac7f7847ce" />

I would like to request that the scrolling feature be reimplemented, and that text boxes sizes should be reverted to what it was previously, like in this example for Panda, where it had this size:

<img width="367" height="96" alt="Image" src="https://github.com/user-attachments/assets/cc86c9a5-cae8-423e-92f7-d6b5726e132b" />

bmcutler commented:
closed as this is a duplicate / issue solved by another PR
